tidb-lightning/tidb-lightning-overview.md

```diff
@@ -47,6 +47,8 @@ TiDB Lightning also supports using TiDB-backend for data import. In this mode, `
 
 ## Restrictions
 
-If you use TiDB Lighting together with TiFlash:
+1. If you use TiDB Lighting together with TiFlash:
 
 No matter a table has TiFlash replica(s) or not, you can import data to that table using TiDB Lightning. Note that this might slow the TiDB Lightning procedure, which depends on the NIC bandwidth on the lightning host, the CPU and disk load of the TiFlash node, and the number of TiFlash replicas.
```

Comment:
```
$ markdownlint tidb-lightning/tidb-lightning-overview.md 
tidb-lightning/tidb-lightning-overview.md: 54: MD029/ol-prefix Ordered list item prefix [Expected: 1; Actual: 2; Style: 1/2/3]
```

```suggestion
    No matter a table has TiFlash replica(s) or not, you can import data to that table using TiDB Lightning. Note that this might slow the TiDB Lightning procedure, which depends on the NIC bandwidth on the lightning host, the CPU and disk load of the TiFlash node, and the number of TiFlash replicas.
```